### ISSUE TITLE
Select volume-template based on content

### ISSUE THREAD

**Issue by GoogleCodeExporter** [braillespecs/obfl#34]
```
Wouldn't it be more logical/uniform if volume-template was a child of 
layout-master? Or alternatively, if layout-master would reference (not contain) 
both (page) templates and volume-templates?

You could have 4 kinds of templates:
- page-template
- default-page-template
- volume-template
- default-volume-template

This would allow you to specify a new volume template for each new sequence. 
For example a "rearmatter" template could be used for the rearmatter. Currently 
you can only specify different volume templates based on the volume number.

Braille CSS has an equivalent feature: see 
http://snaekobbi.github.io/braille-css-spec/#dfn-volume-type-selector and 
http://snaekobbi.github.io/braille-css-spec/#dfn-named-volume.

```

Original issue reported on code.google.com by `bertfrees` on 28 Jan 2015 at 1:11

**Comment by joeha480** [comment]
@bertfrees How can there be more than one volume template per actual volume?

**Comment by bertfrees** [comment]
No, that's not what I mean. Maybe my explanation wasn't that great, but basically what I would like to be able to do is to apply different volume templates for different parts of the document. (How we achieve this in OBFL is less important.) For example, start out with a certain volume template, and for the rearmatter switch to a different one (for example with a different title page). Of course this also implicates that a volume break is forced when the rearmatter begins.

**Comment by joeha480** [comment]
Ok, that seems reasonable, we can think of a solution when looking into more advanced volume breaking.

**Comment by joeha480** [comment]
@bertfrees Does manual volume breaking solve this?

**Comment by bertfrees** [comment]
No. What I was looking for is a feature in OBFL through which I could implement [named volumes](http://braillespecs.github.io/braille-css/20161201/#h3_the-volume-propery) in CSS. Manual volume breaking is just about controlling the break positions, not about the pre/post content.

**Comment by joeha480** [comment]
@bertfrees Sorry, I keep forgetting...
